src/components/Header/Header.tsx

```diff
@@ -9,6 +9,11 @@ import HeaderDescription from './HeaderDescription'
  * @accessibility
  * Headings communicate the organization of the content on the page. Web browsers, plug-ins, and assistive technologies can use them to provide in-page navigation.
  * Nest headings by their rank (or level). The most important heading has the rank 1 (<h1>), the least important heading rank 6 (<h6>). Headings with an equal or higher rank start a new section, headings with a lower rank start new subsections that are part of the higher ranked section.
+ *
+ *
+ * Other considerations:
+ *  - when the description property is used in header, readers will narrate both header content and description within the element.
```

Comment:
could you, please, suggest what this is about? Which case we are describing there (actually, would help if you'll be able to share HTML fragment). Thanks!

Reply:
This is related to the case when header has description, so user would be aware that description is narrated the same way as header
![screen shot 2018-08-22 at 16 42 09](https://user-images.githubusercontent.com/22620150/44470578-68aff780-a62a-11e8-96ef-a64bb011f40f.png)